relatedData.js

```diff
@@ -56,6 +85,27 @@ let deleteTrack = function (id, cb) {
   })
 };
 
-module.exports.saveTrack = saveTrack;
-module.exports.findTrack = findTrack;
-module.exports.deleteTrack = deleteTrack;
\ No newline at end of file
+const drop = () => {
+  mongoose.connection.dropCollection('tracks', (err, result) => {
+    if (err) {
+      console.log('err: ', err);
+    } else {
+      console.log('tracks dropped');
+      console.log(result);
+    }
+  });
+};
+
+const close = () => {
+  mongoose.connection.close();
+};
+
+
+module.exports = {
+  addTrack,
+  findTrack,
+  updateTrack,
+  deleteTrack,
+  drop,
+  close,
+}
```

Comment:
✓ Well done! Looking forward to seeing where the drop and close functions are being called.